app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -227,7 +250,7 @@ $ ->
               if barData.length == 1
                 # Error bar must be present or it causes issues with other bars
                 thisError = {
-                  name: "Error for #{name}"
+                  name: "#{name}"
```

Comment:
bars with only one value have NaN in tooltip for error bars. Needs a value for stdDev.